lib/assesments/assessment_2/fia2milyas/fia2milyas_view/fia2milyas_sign_log/fia2milyas_custom_widget/fia2milyas_button.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+
+class FiA2MIlyasMyAppButton extends StatelessWidget {
+  final String buttontxt;
+  final Color concolor;
+  final Color txtclr;
```

Comment:
buttonText
conColor : PS would be better if you change con with something better cant really understand
textColor